### DISCUSSION TITLE
Deleting the hubitat integration from home assistant

### DISCUSSION THREAD

**Discussion by mrfuntime** [jason0x43/hacs-hubitat#293] [Q&A] [answered]
Is deleting the integration as simple as clicking 'delete' or are there any things I have to do prior to that?

I've retired my hubitat, moving everything directly to home assistant by way of a zigbee coordinator, and would therefore like to delete the Hubitat integration to clean up my setup.

Thanks in advance.

**Comment by jason0x43** [comment, marked as answer]
Yes, just go to Settings -> Integrations -> Hubitat and delete any integration entries.
